fritz-demo-app/app/src/main/java/ai/fritz/heartbeat/Navigation.java

```diff
@@ -56,4 +57,8 @@ public static void goToHairSegmentation(Context context) {
         Intent poseEstimation = new Intent(context, HairSegmentationActivity.class);
         context.startActivity(poseEstimation);
     }
+    public static void goToSkySegmentation(Context context) {
+        Intent poseEstimation = new Intent(context, SkySegmentationActivity.class);
```

Comment:
```Intent poseEstimation``` can you change the name of this instance variable? Also, can you tell me what context refers to in this case (i.e what is the calling context being used for this intent)?